indico/modules/rb/operations/bookings.py

```diff
@@ -363,6 +363,16 @@ def has_same_dates(old_booking, new_booking):
             old_booking.repeat_frequency == new_booking['repeat_frequency'])
 
 
+def has_same_slots(old_booking, new_booking):
+    if not (old_booking.repeat_interval == new_booking['repeat_interval'] and
+            old_booking.repeat_frequency == new_booking['repeat_frequency']):
+        return False
```

Comment:
multi-line if conditions are usually wrapped like this in Python. Also, using or instead of he negation aroudn everything is more readable:

```python
    if (old_booking.repeat_interval != new_booking['repeat_interval'] or
            old_booking.repeat_frequency != new_booking['repeat_frequency']):
        return False
```

or in black-style:

```python
    if (
        old_booking.repeat_interval != new_booking["repeat_interval"]
        or old_booking.repeat_frequency != new_booking["repeat_frequency"]
    ):
        return False
```

Reply:
Going with black style